public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -7,38 +7,47 @@ import {
   rangeUtil,
   RawTimeRange,
   TimeRange,
+  TimeZone,
   toUtc,
 } from '@grafana/data';
-import { DashboardModel } from '../state/DashboardModel';
 import { getShiftedTimeRange, getZoomedTimeRange } from 'app/core/utils/timePicker';
 import { config } from 'app/core/config';
 import { getRefreshFromUrl } from '../utils/getRefreshFromUrl';
 import { locationService } from '@grafana/runtime';
-import { AbsoluteTimeEvent, ShiftTimeEvent, ShiftTimeEventPayload, ZoomOutEvent } from '../../../types/events';
+import { AbsoluteTimeEvent, ShiftTimeEvent, ShiftTimeEventDirection, ZoomOutEvent } from '../../../types/events';
 import { contextSrv, ContextSrv } from 'app/core/services/context_srv';
 import appEvents from 'app/core/app_events';
 
+interface TimeModel {
```

Comment:
One more thing - we need to add `DasboardModel implements TimeModel` to DashboardModel